### ISSUE TITLE
Unable to open CefSharp source code in VS 2015

### ISSUE THREAD

**Issue by nagesh2501** [cefsharp/CefSharp#1705]
Hi,

I am unable to open CefSharp source code in VS 2015. I am getting the following error.

"Project 'CefSharp.BrowserSubprocess.Core' could not be loaded because it's missing install components. To fix this launch Visual Studio setup with the following selections:
 Install Visual C++ 2015 Tools for Windows Desktop"

I am using VS 2015. Also I have Microsoft Visual C++ 2015 Redistributable component installed.

**Comment by amaitland** [comment]
Is it `CefSharp` or the `CefSharp.Tutorial` project that your having problems opening? 

> Also I have Microsoft Visual C++ 2015 Redistributable component installed.

The `Redistributable` is only useful for deploying an app that uses `VC++`, you need to install the `DEV` tools as the issue explicitly says **Install Visual C++ 2015 Tools for Windows Desktop**

**Comment by amaitland** [comment]
@nagesh2501 I also explicitly said ask your questions on `Gitter` and now your opening bug reports here???

**Comment by nagesh2501** [comment]
Dear @amaitland this link is not opening for me. Doesn't look like a valid link.
https://gitter.im/cefsharp/CefSharp#

Also I had posted my query earlier in "CefSharp turotial" link. But my question is supposed to be with CefSharp. 

I am not trying to annoy you or open any bug report.

**Comment by amaitland** [comment]
> Doesn't look like a valid link.
> https://gitter.im/cefsharp/CefSharp#

What do you mean by that? Have you opened it? Or your just looking at it and it looks different? It's the chat channel where people discuss `CefSharp` support questions. They also have an app for `Android` and `iOS`, all you need is a `GitHub` account which you quite clearly already have.

> I am not trying to annoy you

You **are** annoying me by not following directions. I understand you think your project is important and that's fine. You need to remember that this is open source, I'm contributing **my own time** to helping you, nobody is paying me, lots of people don't even say **thank you**. The least you can do is follow a few simple instructions, is it that hard??? If that doesn't work for you then look at using one of the other `CEF` based wrappers.

> or open any bug report.

You opened three in a matter of minutes, two identical. The third being this one, quite clearly indicates your missing components from `VS`, you just didn't take the time to read over the error message.

**Comment by amaitland** [comment]
I think it might be better if you looked at using one of the other `CEF` wrappers, checkout `CefGlue` or `ChromiumFx`. Good luck!

**Comment by nagesh2501** [comment]
![gitter](https://cloud.githubusercontent.com/assets/19825195/15947647/47ae35ca-2e62-11e6-99d3-0366d67b93b4.JPG)


Hey.. I am really sorry that you got annoyed. I really didn't intend to. I am just new to this place. So in the first instance I have opened a bug instead of posting a question. I did not realize that. I understand your pain. 

This link is really not opening in my browser. Also checked at my peer's desk.
https://gitter.im/cefsharp/CefSharp#
